scheduler/src/main/java/org/pentaho/platform/scheduler2/ws/ISchedulerService.java

```diff
@@ -104,4 +104,6 @@ public String updateJobComplexTriggerWithJobName( String jobName, String jobId,
    */
   public int getSchedulerStatus() throws SchedulerException;
 
+  public boolean canStopScheduler();
```

Comment:
Adding a new method to an Interface may be "dangerous"...
We have to check this better...

Reply:
Added default behaviour for interface mmethod